### ISSUE TITLE
Multiplayer text editor duplicating initial content

### ISSUE THREAD

**Issue by henriquedevelops** [liveblocks/liveblocks#1377]
I'm working on a web app that already has a create notepads functionality with Tiptap formatting. The idea now is to make these notepads collaborative.

I managed to implement the collaboration functionality using Liveblocks and Yjs.

The problem is that, when I open an already existing notepad with the now collaborative text editor, the text content is lost (it shows up a blank notepad).

So I tried passing the initial content to the useEditor hook settings like this:

```
  const editor = useEditor({
   // ...other settings
    content: initialContent,
  });
```

This works on the first time that I open an already existing notepad: I open it and the initial text content is there.

However, it keeps adding this initial content to the notepad every time that I open this notepad, as shown in the video below:

https://github.com/liveblocks/liveblocks/assets/47286998/a3297eb8-efda-48ed-9ca4-8ab1e3a42ad5

Basically what it does is: it concatenates the initial text with the text stored in the Yjs document every time that I open the notepad.

To better illustrate this, I've changed the initial value (retrieved from the database) to a string "initial value test", like this:

```
  const editor = useEditor({
   // ...other settings
    content: "initial value test",
  });
```

https://github.com/liveblocks/liveblocks/assets/47286998/e1c22f07-1e4e-4c6e-bbab-47364e30d650
